### PR TITLE
CDDP Subcategorization

### PR DESCRIPTION
**Pull request by Daphne210**

### CODE REVIEW

**Comment by slubwama** [comment]
@Daphne210 resolve conflicts

**Comment by Daphne210** [comment]
Conflicts have been resolved

File: api/src/main/resources/liquibase.xml [hunk 1]
```diff
@@ -628,6 +628,14 @@ VALUES
...
 		</sql>
 	</changeSet>
+  
+	<changeSet id="ugandaemr-20230127-2345" author="daphinenalule">
+		<comment>
+			Insert General CDDP Cohort type for CDDP Group
+		</comment>
+		<sql>
+			REPLACE INTO cohort_type(name,date_created,creator,uuid)VALUES
+			('General CDDP Cohort Type','2023-01-27',1,'da964ff0-648e-440b-a219-d2aeba3670d0');
```
Comment (dbaluku) [inline api/src/main/resources/liquibase.xml:638]: You have not closed tags here

File: omod/src/main/webapp/resources/htmlforms/HMIS-HIV-003-HivCareArtCard-ClinicalAssessmentPage.xml [hunk 2]
```diff
@@ -1606,6 +1611,28 @@
...
+                                 }
+                             },
+                            error: function(jqXHR, textStatus, errorThrown) {
+                                  getField('returndate.error').html('Unable to determine if ' + formattedDate + ' is a public Holiday').show();
+                                    jq().toastmessage('showErrorToast', 'Unable to determine if ' + formattedDate + ' is a public Holiday');
+                                    isValid  = false;
+                                }
+                        });
+
+                    }
```
Comment (dbaluku) [inline omod/src/main/webapp/resources/htmlforms/HMIS-HIV-003-HivCareArtCard-ClinicalAssessmentPage.xml:1634]: where is this coming, I see it is repeated below and its not part of out work

File: omod/src/main/webapp/resources/htmlforms/HMIS-HIV-003-HivCareArtCard-ClinicalAssessmentPage.xml [hunk 3]
```diff
@@ -1684,6 +1711,13 @@
                     return false;
                 }
 
+            </restrictByRole>
```
Comment (dbaluku) [inline omod/src/main/webapp/resources/htmlforms/HMIS-HIV-003-HivCareArtCard-ClinicalAssessmentPage.xml:1691]: this section is also repeated

File: omod/src/main/webapp/resources/htmlforms/HMIS-HIV-003-HivCareArtCard-ClinicalAssessmentPage.xml [hunk 4]
```diff
@@ -3139,6 +3173,7 @@
                                                 <div class="row">
                                                     <div class="col-md-3" id="cddp_category">
                                                         <label>CDDP Category</label>
+                                                        <workflowState  workflowId="4" stateIds="16,14,15" stateLabels="General CDDP, CLDDP, CRPDDP"/>
```
Comment (dbaluku) [inline omod/src/main/webapp/resources/htmlforms/HMIS-HIV-003-HivCareArtCard-ClinicalAssessmentPage.xml:3151]: this is also repeated

File: omod/src/main/webapp/resources/htmlforms/HMIS-HIV-003-HivCareArtCard-ClinicalAssessmentPage.xml (hunk 3 above)
Comment (dbaluku) [reply omod/src/main/webapp/resources/htmlforms/HMIS-HIV-003-HivCareArtCard-ClinicalAssessmentPage.xml:1691]: This section is still repeated just below it

File: omod/src/main/webapp/resources/htmlforms/HMIS-HIV-003-HivCareArtCard-ClinicalAssessmentPage.xml (hunk 4 above)
Comment (dbaluku) [reply omod/src/main/webapp/resources/htmlforms/HMIS-HIV-003-HivCareArtCard-ClinicalAssessmentPage.xml:3151]: this section is still repeated here too

**Pull request closed without merging**